tdp/core/collection.py

```diff
@@ -3,10 +3,14 @@
 
 from pathlib import Path
 
+from networkx.readwrite.json_graph.jit import json
```

Comment:
Why not use `import json` ?

Reply:
Oups, automatic import from my ide, it's fixed